Source/AliveLibAE/Movie.cpp

```diff
@@ -311,7 +314,8 @@ EXPORT char CC DDV_Play_Impl_4932E0(const char* pMovieName)
         return 0;
     }
 
-    while (Input_IsVKPressed_4EDD40(VK_ESCAPE) || Input_IsVKPressed_4EDD40(VK_RETURN))
+    while (Input_IsVKPressed_4EDD40(VK_ESCAPE) || Input_IsVKPressed_4EDD40(VK_RETURN) ||
+        Input_Read_Pad_4FA9C0(sCurrentControllerIndex_5C1BBE) & MOVIE_SKIPPER_GAMEPAD_INPUTS)
```

Comment:
should only be called if game pad is active? does it still make sense to check both keyboard + pad? 🤔